addons/mail/static/src/discuss/call/common/rtc_service.js

```diff
@@ -272,7 +272,7 @@ export class Rtc extends Record {
             connectionType: undefined,
             hasPendingRequest: false,
             channel: undefined,
-            globalLogs: {},
+            globalLogs: { version: odoo.info?.server_version },
```

Comment:
Can't we dump the whole `odoo.info`? There is more information like the database name and `isEnterprise`. Maybe it can be handy at some point?

Reply:
@phenix-factory updated